scripts/run_e2e_tests.py

```diff
@@ -434,18 +434,32 @@ def get_chrome_driver_version():
     This method follows the steps mentioned here:
     https://chromedriver.chromium.org/downloads/version-selection
     """
+    popen_args = ['google-chrome', '--version']
+    command_that_errors = 'google-chrome --version'
```

Comment:
Don't have two sources of truth. Derive command_that_errors inline from`' '.join(popen_args)` and get rid of that latter variable altogether.

That will allow you to replace this and the following with a single:

`popen_args = ... if common.is_mac_os() else ...`

Reply:
I kept that latter variable because there were backslashes in it to escape the spaces in the path. I think it's more clear to keep it as a separate variable. I added comments in to be more clear about this. What do you think?

Reply:
My comment still applies, but I understand your concern. Perhaps you can handle this by something like `' '.join(arg.replace(' ', '\ ') for arg in popen_args)` or similar. (Still keep the comments, in order to explain why we're doing this.)

Reply:
Okay! I made the change - the arg.replace seems like a nice way to help with the readability.